app/uk/gov/hmrc/merchandiseinbaggage/model/api/JourneyDetails.scala

```diff
@@ -17,19 +17,24 @@
 package uk.gov.hmrc.merchandiseinbaggage.model.api
 
 import java.time.LocalDate
-import java.time.format.DateTimeFormatter
 
-import play.api.libs.json.{JsObject, JsResult, JsSuccess, JsValue, Json, OFormat}
+import play.api.libs.json._
 import uk.gov.hmrc.merchandiseinbaggage.model.api.YesNo.{No, Yes}
 
 sealed trait JourneyDetails {
   val port: Port
   val dateOfTravel: LocalDate
-  val formattedDateOfArrival: String = DateTimeFormatter.ofPattern("dd MMM yyyy").format(dateOfTravel)
```

Comment:
not sure about this, have you tested E2E locally?

Reply:
nope, but when doing find usage => 0 result and also find in project => 0 result.....guess should be fine

Reply:
my suspicion is that it's used in the frontend to display the date on CheckYourAnswers. Will need to be moved out of the case class in the frontend and add a method to format it to JourneyDetailsEnriched maybe? You may have already done this